pkg/services/ngalert/provisioning/alert_rules.go

```diff
@@ -75,10 +78,6 @@ func (service *AlertRuleService) CreateAlertRule(ctx context.Context, rule model
 		} else {
 			return errors.New("couldn't find newly created id")
 		}
-		err = service.ruleStore.UpdateRuleGroup(ctx, rule.OrgID, rule.NamespaceUID, rule.RuleGroup, rule.IntervalSeconds)
```

Comment:
When we create a rule, we take the default interval if it's the first rule in the group or we fetch the current group interval. Not sure why I thought we need to update the rule group. Removed it.